schema/point.go

```diff
@@ -13,6 +13,18 @@ type PointSchema struct {
 
 	IoType schema.IoType `json:"io_type"`
 
+	AddressId struct {
+		Type        string `json:"type" default:"number"`
+		Title       string `json:"title" default:"Point Id"`
+		Default     int    `json:"default" default:"1"`
+		Minimum     int    `json:"minimum" default:"1"`
+		Maximim     int    `json:"maximum" default:"9999"`
```

Comment:
> Point ID - 8 Bits - -> optional on positional flag otherwise auto inferred and incremented
```
		Maximim     int    `json:"maximum" default:"256"`
```

Reply:
There is a typo as well. 😄